tests/unit/generator/lib.php

```diff
@@ -200,4 +202,29 @@ public function enrol_test_user($moodle_user, $course, $role) {
         $instance = $DB->get_record("enrol", array('courseid' => $course, 'enrol' => 'manual'));
         $enrol->enrol_user($instance, $moodle_user, $roleid);
     }
+
+    /**
+     * Create a test submission.
+     *
+     * @param $turnitintooltwoassignment
+     * @param $author
+     * @param $partid
+     */
+    public function create_test_submission($turnitintooltwoassignment, $author, $partid) {
```

Comment:
I think this should go in `/generator/lib.php` as we have functions in there for generating assignments/parts etc.

Reply:
That file isn't used. The `tests/unit/generator/lib.php` was written a while back and is used instead.

Reply:
Ah, that's actually the file I meant with my file path. I missed the other `generator/lib.php`.

In that case shouldn't we delete `/tests/generator/lib.php` and put this function in `tests/unit/generator/lib.php`?

Reply:
I'll remove `/tests/generator/lib.php`